packages/frontend-core/src/components/grid/controls/ColumnsSettingButton.svelte

```diff
@@ -1,9 +1,12 @@
 <script>
   import { getContext } from "svelte"
-  import { ActionButton, Popover, Icon } from "@budibase/bbui"
+  import { ActionButton, Popover, Icon, notifications } from "@budibase/bbui"
   import { getColumnIcon } from "../lib/utils"
   import ToggleActionButtonGroup from "./ToggleActionButtonGroup.svelte"
 
+  export let showReadonlyColumnsOptions = false
```

Comment:
I don't like this double booleans, but I think it's cleaner than using enums in JS or complex objects as parameters..

Reply:
Yea I agree it's probably better to just use an explicit boolean for whether or not we can use views, rather than passing around a license prop or something and duplicating the logic in here.